src/index.ts

```diff
@@ -10,9 +10,10 @@ require('dotenv').config();
 
 export const client: Client = new Client();
 
-const USE_NEW_COMMAND_LOADER = false;
+const USE_NEW_COMMAND_LOADER = true;
 
 async function init() {
+  await BindDatabase();
```

Comment:
commands were loading that depended on the database being ready. so moved here to ensure ready.